pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
```

Comment:
Please update module version to next major version

Reply:
Done